owl2nl/src/main/java/org/aksw/owl2nl/OptimizerDepParse.java

```diff
@@ -0,0 +1,239 @@
+package org.aksw.owl2nl;
+
+import edu.stanford.nlp.ling.IndexedWord;
+import edu.stanford.nlp.pipeline.CoreDocument;
+import edu.stanford.nlp.pipeline.CoreSentence;
+import edu.stanford.nlp.pipeline.StanfordCoreNLP;
+import edu.stanford.nlp.semgraph.SemanticGraph;
+import edu.stanford.nlp.semgraph.SemanticGraphFormatter;
+//import org.graalvm.compiler.core.common.type.ArithmeticOpTable;
```

Comment:
Are you keeping this line commented for testing in the future?
If so, please specify that as a comment above it.